plugins/firebase.js

```diff
@@ -13,6 +14,7 @@ if (!firebase.apps.length) {
 }
 
 export const db = firebase.database()
+export const firestoreDb = firebase.firestore()
```

Comment:
Rename this variable as `DB`, this would be the name everyone will proceed with

Reply:
Sure!